activiti-cloud-service-common/activiti-cloud-services-test/src/main/java/org/activiti/cloud/starters/test/builder/ProcessInstanceEventContainedBuilder.java

```diff
@@ -61,4 +62,14 @@ private ProcessInstanceImpl buildProcessInstance(String name) {
         return completedProcess;
     }
 
+    public ProcessInstanceImpl aRunningProcessInstanceWithStartDate(String name, Date startDate) {
+        ProcessInstanceImpl processInstance = buildProcessInstance(name);
+        processInstance.setStartDate(startDate);
+        eventsAggregator.addEvents(new CloudProcessCreatedEventImpl(processInstance),
+            new CloudProcessStartedEventImpl(processInstance,
```

Comment:
Doesn't `CloudProcessStartedEventImpl` have a constructor that takes only process instance as parameter to avoid extra `null`s?

Reply:
Yes, good point.